manual/source/defs_intro.rst

```diff
@@ -57,11 +57,62 @@ General documentation if this NXDL file.
 Symbols table
 =============
 
-The symbols table describes 
-keywords used in this NXDL file to designate array dimensions. 
-At present, this list is not guaranteed to be complete 
-(some array dimension names appear only in a *Structure* 
-description and not here).
+The ``Symbols`` table describes keywords used in this NXDL file to designate 
+array dimensions. For reasons of avoiding naming collisions and to facilitate 
+readbility and comprehension for those whom are new to an NXDL file, the following
+guidelines are strongly encouraged:
+
+* All symbols used in the application definition are defined in a single ``Symbols`` table.
+* The :ref:`name <validItemName>` of a symbol uses camel case without any white space or underscores.
+
+  examples: 
+
+ 		**nP**: Total number of scan points
+
+ 		**nE**: Number of photon energies scanned
+
+ 		**nFrames**: Number of frames
+
+ 		**detectorRank**: Rank of data array provided by the detector for a single measurement
+
+* the ``Symbols`` table appears early in the .nxdl file above the ``NXentry`` group
+
+ example from `NXtomo.nxdl.xml <https://github.com/nexusformat/definitions/blob/main/applications/NXtomo.nxdl.xml>`_
+
+.. code-block:: xml	
```

Comment:
Could try a [literalinclude](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html?highlight=code-block#directive-literalinclude) instead but the path to the file gets a bit scary.